src/App.js

```diff
@@ -1,16 +1,18 @@
 import React from "react";
-import logo from "./logo.png";
 import "./App.css";
+import logo from "./logo.png";
+import Clock from "./Clock.js";
 
 class App extends React.Component {
   render() {
     return (
       <div className="App">
         <header className="App-header">
           <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
+          <p>Singapore: {<Clock timeZone="Asia/Singapore" />}</p>
```

Comment:
your `Clock` component is actually a div with some text in it. so you shouldn't put it within another `p` tag. better if the clock is _beside_ the `p` rather than _in_ the `p`